tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/ManagedIdentityTests.NetFwk.cs

```diff
@@ -285,6 +355,109 @@ public async Task AcquireMsiToken_ExchangeForEstsToken_Successfully()
             }
         }
 
+        [TestMethod]
+        public async Task AcquireMsiToken_ExchangeForEstsTokenWithClaims_Successfully()
```

Comment:
Same comment for this one - you are only bypassing the cache. Unit test is sufficient.